_includes/scenarios/integrate/establish-a-relationship-to-another-identity.md

```diff
@@ -0,0 +1,164 @@
+
+```jsonc
+{
+  "templateId": "<ID of RelationshipTemplate>",
+  "content": {
+    // Customized content
+    ...
+  }
+}
+```
+
+Note that the `content` property is optional and can therefore be omitted. In case of success, you will receive a `result` as response, which contains the [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) and in particular an associated [RelationshipChange]({% link _docs_integrate/data-model-overview.md %}#relationshipchange). The RelationshipChange has `"Creation"` as `type` and `"Pending"` as `status`. If you have specified a customized content within the `content` property of the input, it is contained within the `request.content` property of the RelationshipChange.
+
+Keeping the `id` of the Relationship and the `id` of the associated RelationshipChange in mind is useful if you want to return to the created Relationship later in order to retrace changes to the Relationship.
+{: .notice--info}
+
+### Send it as an App user
+
+As already mentioned in the description of the [input for creating a RelationshipTemplate]({% link _docs_integrate/establish-a-relationship-to-another-identity.md %}#input-for-creating-a-relationshiptemplate), a RelationshipTemplateContent must be used as the value of the `content` property of the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) if you intend to use the RelationshipTemplate to establish a Relationship between the Templator and an App user. Assuming that there is no Relationship between them yet, the App user receives the Request specified in the `onNewRelationship` property of the [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) after scanning the QR Code associated with the RelationshipTemplate. The App user has the option of accepting or rejecting the Request. If they accept the Request appropriately, a Relationship Request is sent to the Templator. Internally, this corresponds to the creation of a data object of type [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) with an associated RelationshipChange and `"Pending"` as `status`. The [RelationshipChange]({% link _docs_integrate/data-model-overview.md %}#relationshipchange) has `"Creation"` as `type` and `"Pending"` as `status`. The `request.content.response` property of the RelationshipChange contains the Response of the App user to the Request.
```

Comment:
I looks like you were mentioning here twice, that the RelationshipChange has `"Pending"` as `status`. Only reading it again, I see that one time you refer to the status of the Relationship and the other time to the status of the RelationshipChange. Maybe you can make this clearer.